sound/soc/amd/acp/soc_amd_sdw_common.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: GPL-2.0-only
+ *  Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved
+ */
+
+/*
+ *  soc_amd_sdw_common.h - prototypes for common helpers
+ */
+
+#ifndef SOC_AMD_SDW_COMMON_H
+#define SOC_AMD_SDW_COMMON_H
+
+#include <linux/bits.h>
+#include <linux/types.h>
+#include <sound/soc.h>
+#include <sound/soc_sdw_utils.h>
+
+#define SDW_MAX_CPU_DAIS	8
+#define SDW_MAX_LINKS		2
+
+#define SDW_MAX_GROUPS 9
+
+#define ACP63_PCI_REV		0x63
+#define SOC_JACK_JDSRC(quirk)		((quirk) & GENMASK(3, 0))
+#define ASOC_SDW_FOUR_SPK		BIT(4)
+#define ASOC_SDW_ACP_DMIC		BIT(5)
+
+#define AMD_SDW0	0
+#define AMD_SDW1	1
+#define SW0_AUDIO0_TX	0
+#define SW0_AUDIO1_TX	1
+#define SW0_AUDIO2_TX	2
+
+#define SW0_AUDIO0_RX	3
+#define SW0_AUDIO1_RX	4
+#define SW0_AUDIO2_RX	5
+
+#define SW1_AUDIO0_TX	0
+#define SW1_AUDIO0_RX	1
```

Comment:
the prefixes are inconsistent, should be SDW_AMD or something.

Reply:
As this header file itself , AMD specific, I don't think we need to follow prefix convention here

Reply:
we will change the prefix convention with ACP as prefix, because these macros may vary based on ACP version.